build/azure-pipelines/win32/product-build-win32.yml

```diff
@@ -89,6 +89,38 @@ steps:
     condition: and(succeeded(), ne(variables.NODE_MODULES_RESTORED, 'true'), ne(variables['NPM_REGISTRY'], 'none'))
     displayName: Setup NPM Authentication
 
+  - powershell: |
+      $includes = @'
+        {
+          'target_defaults': {
+            'conditions': [
+              ['OS=="win"', {
+                'msvs_configuration_attributes': {
+                  'SpectreMitigation': 'Spectre'
+                },
+                'msvs_settings': {
+                  'VCCLCompilerTool': {
+                    'AdditionalOptions': [
+                      '/ZH:SHA_256'
+                    ],
+                  },
+                  'VCLinkerTool': {
+                    'AdditionalOptions': [
+                    ]
+                  }
+                }
+              }]
+            ]
+          }
+        }
+      '@
+
+      if (!(Test-Path "~/.gyp")) {
+        mkdir "~/.gyp"
+      }
+      echo $includes > "~/.gyp/include.gypi"
```

Comment:
Why is this re-introduced ? I thought we added the flags to all the modules that needed it in their respective repo.

Reply:
I copied it over from the SDL pipeline and thought it was a good catch-all in case an upstream package didn't have those flags yet. I can remove it so that we can add the config to upstream repositories that need it.